src/interface/meta.thrift

```diff
@@ -43,10 +46,19 @@ enum AlterSchemaOp {
 } (cpp.enum_strict)
 
 
+enum RoleType {
+    GOD    = 0x01,
```

Comment:
Add some comments, what's the meaning for different roles?

Reply:
Good suggest!